### PR TITLE
reference: update parameter description in loader template

### PR DESCRIPTION
**Pull request by dcalvin**
### What is changed, added or deleted? 

This PR updated some parameter description in the loader config template

### What is the related PR or file link(s)? 

 https://github.com/pingcap/docs-cn/pull/1906 

### Which version does your change affect? 

TiDB 3.0/2.1/Dev

### CODE REVIEW

File: dev/reference/tools/loader.md [hunk 1]
```diff
@@ -97,12 +97,9 @@ user = "root"
...
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream will be used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
+# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If set to 0, the global `max-allowed-packet` for downstream will be used.
```
Comment (amyangfei) [inline dev/reference/tools/loader.md]:
```suggestion
# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max_allowed_packet` in system parameters. If set to 0, the global `max_allowed_packet` for downstream will be used.
```
this variable in MySQL/TiDB is `max_allowed_packet`, the same for v3.0, v2.1

File: dev/reference/tools/loader.md [hunk 2]
```diff
@@ -97,13 +97,10 @@ user = "root"
...
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream will be used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
-max-allowed-packet = 67108864
+# `max_allowed_packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If set to 0, the global `max-allowed-packet` for downstream will be used.
+max_allowed_packet = 67108864
```
Comment (amyangfei) [inline dev/reference/tools/loader.md]: oh this is misused, `max-allowed-packet` is used in this config template, and `max_allowed_packet` is the system parameter in the database

Comment (lilin90) [inline dev/reference/tools/loader.md]:
```suggestion
max-allowed-packet = 67108864
```

File: dev/reference/tools/loader.md [hunk 3]
```diff
@@ -97,13 +97,10 @@ user = "root"
...
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream will be used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
-max-allowed-packet = 67108864
+# `max_allowed_packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If set to 0, the global `max-allowed-packet` for downstream will be used.
```
Comment (lilin90) [inline dev/reference/tools/loader.md]:
```suggestion
# `max_allowed_packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If it is set to 0, the global `max-allowed-packet` for downstream is used.
```

File: dev/reference/tools/loader.md [hunk 4]
```diff
@@ -97,13 +97,10 @@ user = "root"
 password = ""
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream will be used.
```
Comment (lilin90) [inline dev/reference/tools/loader.md]:
```suggestion
# `sql_mode` of the session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream is used.
```

Comment (dcalvin) [reply dev/reference/tools/loader.md]: > of the session level 

I am not sure about this one

**Comment by dcalvin** [comment]
PTAL again @amyangfei @lilin90  Thanks.

File: v3.0/reference/tools/loader.md [hunk 5]
```diff
@@ -98,12 +98,9 @@ user = "root"
...
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream is used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
+# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If it is set to 0, the global `max-allowed-packet` for downstream is used.
```
Comment (amyangfei) [inline v3.0/reference/tools/loader.md]:
```suggestion
# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max_allowed_packet` in system parameters. If it is set to 0, the global `max_allowed_packet` for downstream is used.
```

File: v2.1/reference/tools/loader.md [hunk 6]
```diff
@@ -97,12 +97,9 @@ user = "root"
...
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream is used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
+# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If it is set to 0, the global `max-allowed-packet` for downstream is used.
```
Comment (amyangfei) [inline v2.1/reference/tools/loader.md]:
```suggestion
# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max_allowed_packet` in system parameters. If it is set to 0, the global `max_allowed_packet` for downstream is used.
```

File: dev/reference/tools/loader.md [hunk 7]
```diff
@@ -97,12 +97,9 @@ user = "root"
...
 port = 4000
 
-# `sql_mode` used when loading data. If the `sql-mode` value is not provided or set to "@DownstreamDefault", downstream global `sql_mode` will be used
+# `sql_mode` of session level used to connect to the database when loading data. If `sql-mode` is not provided or set to "@DownstreamDefault", the global `sql_mode` for downstream is used.
 # sql-mode = ""
-
-# max-allowed-packet sets `max_allowed_packet` for database connection.
-# `max-allowed-packet=0` can be used to automatically fetch the
-# `max_allowed_packet` variable from server on every connection.
+# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max-allowed-packet` in system parameters. If it is set to 0, the global `max-allowed-packet` for downstream is used.
```
Comment (amyangfei) [inline dev/reference/tools/loader.md]:
```suggestion
# `max-allowed-packet` sets the maximum data packet allowed for database connection, which corresponds to the `max_allowed_packet` in system parameters. If it is set to 0, the global `max_allowed_packet` for downstream is used.
```

**Comment by dcalvin** [comment]
Thanks. @amyangfei PTAL again.

**Pull request merged**